token-swap/js/client/token-swap.js

```diff
@@ -93,11 +98,14 @@ type TokenSwapInfo = {|
  */
 const TokenSwapLayout = BufferLayout.struct([
   BufferLayout.u8('state'),
+  BufferLayout.u8('nonce'),
   Layout.publicKey('tokenAccountA'),
   Layout.publicKey('tokenAccountB'),
   Layout.publicKey('tokenPool'),
+  BufferLayout.blob(7, 'padding'),
```

Comment:
Oof, I'm looking forward to the packing rework to get this out of here :)

Reply:
Ditto. Testing these structures was such a headache